src/main/java/de/tum/in/www1/artemis/domain/notification/GroupNotificationFactory.java

```diff
@@ -123,20 +161,24 @@ public static GroupNotification createNotification(Exercise exercise, User autho
 
         // Catches 3 different use cases : notificationText exists for 1) non-exam exercise update & 2) live exam exercise update with a notification text
         // 3) hidden/silent live exam exercise update without a set notificationText, thus no pop-up will be visible for the students
-        if (notificationText != null || LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE.equals(title)) {
+        if (notificationText != null || NotificationConstants.LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE.equals(title)) {
             text = notificationText;
+            textIsPlaceholder = false;
+            placeholderValues = new String[] { exercise.getCourseViaExerciseGroupOrCourseMember().getTitle() };
         }
 
-        GroupNotification notification = new GroupNotification(exercise.getCourseViaExerciseGroupOrCourseMember(), title, text, author, groupNotificationType, priority);
+        GroupNotification notification = new GroupNotification(exercise.getCourseViaExerciseGroupOrCourseMember(), title, text, textIsPlaceholder, placeholderValues, author,
+                groupNotificationType, priority);
 
         // Exercises for exams
         if (exercise.isExamExercise()) {
-            if (LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE.equals(title)) {
+            if (NotificationConstants.LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE.equals(title)) {
                 notification.setTransientAndStringTarget(createExamExerciseTargetWithExerciseUpdate(exercise));
                 notification.setPriority(HIGH);
             }
             else if (exercise instanceof ProgrammingExercise) {
-                notification.setTransientAndStringTarget(createExamProgrammingExerciseOrTestCaseTarget((ProgrammingExercise) exercise, EXERCISE_UPDATED_TEXT));
+                notification.setTransientAndStringTarget(
+                        createExamProgrammingExerciseOrTestCaseTarget((ProgrammingExercise) exercise, NotificationTargetFactory.EXERCISE_UPDATED_TEXT));
```

Comment:
Sadly GitHub does not allow suggestions in non edited lines:

else if (exercise instanceof ProgrammingExercise programmingExercise) {
                notification.setTransientAndStringTarget(createExamProgrammingExerciseOrTestCaseTarget(programmingExercise, EXERCISE_UPDATED_TEXT));
                notification.setTransientAndStringTarget(
                        createExamProgrammingExerciseOrTestCaseTarget(programmingExercise, NotificationTargetFactory.EXERCISE_UPDATED_TEXT));